src/Analysis/Ast/Impl/Modules/PythonModule.cs

```diff
@@ -244,6 +244,7 @@ internal PythonModule(ModuleCreationOptions creationOptions, IServiceContainer s
             _diagnosticsService?.Remove(Uri);
             _allProcessingCts.Cancel();
             _allProcessingCts.Dispose();
+            Services.GetService<IPythonAnalyzer>().RemoveAnalysis(this);
```

Comment:
Can we use `DisposeToken` here instead of `_allProcessingCts`?

Reply:
Sure